ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1746,6 +1746,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-stack-ipi
+- as: azure-stack-ipi-proxy-p2-f7
```

Comment:
Maybe we can lower test frequency of `azure-stack-ipi` a bit to `f28` based on `azure-stack-ipi-proxy` almost can cover the common ipi install.

Reply:
agree, updated.